src/screens/Books/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import { connect } from 'react-redux';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+
+import { actionCreators as bookActions } from '../../redux/books/actions';
+import { Spinner } from '../../components/Spinner';
+import { ErrorModal } from '../../components/ErrorModal';
```

Comment:
Use default export in components `Spinner` and `ErrorModal`.

Reply:
Done on the next commit.